schematics/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="grid-container">
+  <h1 class="mat-h1">Dashboard</h1>
+  <mat-grid-list cols="2" rowHeight="350px">
```

Comment:
Grid-list is only useful when you want items with heterogenous rowspans. We could either mix that in, or just use cards laid out with flex styles.

Reply:
I'll update it to have spans :)

Reply:
Are the rowspans meaningful here? A screenshot would be helpful